configs/metadata.yaml

```diff
@@ -133,5 +167,20 @@ linz:
       red: 25
     wavelength:
       blue: 0.48
-      green: 0.53
-      red: 0.66
+      green: 0.56
+      red: 0.65
+sentinel-1-rtc:
+  band_order:
+    - vv
+    - vh
+  gsd: 10
```

Comment:
Very good point. Maybe this could be renamed to pixel spacing. I think this is what the model need to know the most here, what size a single pixel represents. The GSD would be good additional information, but it would have to be specified per band for multiple systems. For instance, for Sentinel-2 we resample everything to a pixel spacing of 10m, although some bands have 20m GSD. So lets keep it as is for now and update naming later?

Reply:
Sure, we can keep the naming for now (otherwise the model code needs to be updated too).